tools/XCConfigExporter/main.swift

```diff
@@ -0,0 +1,66 @@
+import XcodeCompilationDatabaseCore
+import Foundation
+import ShellOut
+
+guard CommandLine.arguments.count > 1 else {
+    print("""
+          usage: /path/to/xcconfig
+          """)
+    exit(0)
+}
+
+/// This program works by doing a build of an iOS application with a user
+/// specified xcconfig, and exporting the flags that Xcode passed to various
+/// compilers
+do {
+    guard let fixtureBundle = Bundle.main.path(forResource: "Fixtures", ofType:
+                                             "bundle") else {
+        fatalError("Missing fixtures")
+    }
+    let xcconfig = CommandLine.arguments[1]
```

Comment:
You should `guard` and return a good error message if an xcconfig isn't passed

Reply:
Will do ✅